VMEncryption/main/EncryptionSettingsUtil.py

```diff
@@ -14,7 +14,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import httplib
+import http.client
```

Comment:
try except for python2

Reply:
http.client should work equally if python2 environment has been prepped with pip install future, added to requirements.txt for now